### PR TITLE
Pipe: add ops/latency metrics for different types of pipe operations on receivers (DN / CN)

### PR DESCRIPTION
**Pull request by YC27**
## Description

add ops monitoring metrics for confignode and datanode to measure different pipe receivers
the four indicators are as follows:
1. OPS Of PipeReceiver
2. Time Consumed Of PipeReceiver (avg)
3. Time Consumed Of PipeReceiver (50%)
4. Time Consumed Of PipeReceiver (99%)

the effect is as follows:
#### DataNode
image

#### ConfigNode
image

### CODE REVIEW

File: iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/manager/pipe/metric/PipeConfigNodeReceiverMetrics.java [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+import org.apache.iotdb.metrics.metricsets.IMetricSet;
+import org.apache.iotdb.metrics.type.Timer;
+import org.apache.iotdb.metrics.utils.MetricLevel;
+import org.apache.iotdb.metrics.utils.MetricType;
+
+public class PipeConfigNodeReceiverMetrics implements IMetricSet {
+
+  private static final PipeConfigNodeReceiverMetrics INSTANCE = new PipeConfigNodeReceiverMetrics();
+
+  private Timer handShakeConfigNodeV1Timer = IoTDBMetricManager.getInstance().createTimer();
```
Comment (Caideyipi) [inline iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/manager/pipe/metric/PipeConfigNodeReceiverMetrics.java:16]: "Handshake" is a single word here.

File: iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/manager/pipe/metric/PipeConfigNodeReceiverMetrics.java [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+            MetricLevel.IMPORTANT,
+            Tag.NAME.toString(),
+            RECEIVER,
+            Tag.TYPE.toString(),
+            "transferConfigSnapshotSeal");
+  }
+
+  @Override
+  public void unbindFrom(AbstractMetricService metricService) {
+    unBindToTimer(metricService);
```
Comment (Caideyipi) [inline iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/manager/pipe/metric/PipeConfigNodeReceiverMetrics.java:100]: If you use this style, it's unbindFromTimer.. BTW, personally I deem it better to use "unbind" to be consistent with the dataNode and other module's style. Besides, unbind is a single word too.

File: iotdb-core/datanode/src/main/java/org/apache/iotdb/db/pipe/metric/PipeDataNodeReceiverMetrics.java [hunk 3]
```diff
@@ -0,0 +1,315 @@
...
+                    MetricLevel.IMPORTANT,
+                    Tag.NAME.toString(),
+                    RECEIVER,
+                    Tag.TYPE.toString(),
+                    "transferSchemaSnapshotSeal");
+  }
+
+  @Override
+  public void unbindFrom(AbstractMetricService metricService) {
+    unBindTimer(metricService);
```
Comment (Caideyipi) [inline iotdb-core/datanode/src/main/java/org/apache/iotdb/db/pipe/metric/PipeDataNodeReceiverMetrics.java:201]: unbind

File: iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/manager/pipe/metric/PipeConfigNodeReceiverMetrics.java (hunk 1 above)
Comment (Caideyipi) [reply iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/manager/pipe/metric/PipeConfigNodeReceiverMetrics.java:16]: Besides, use "DoNothingTimer()" in initialization and "unbind"...

Comment (Caideyipi) [reply iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/manager/pipe/metric/PipeConfigNodeReceiverMetrics.java:16]: Besides, use "DoNothingTimer()" in initialization and "unbind"...

**Review by SteveYurongSu — APPROVED** [review]

**Pull request merged**